### DISCUSSION TITLE
How to modify the content width of my Jupyter book

### DISCUSSION THREAD

**Discussion by quantitative-technologies** [executablebooks/meta#797] [Q&A]
I want to make the middle content column wider, and also make the total page width larger since it only uses less than 2/3 of the screen width when I read the book in my web browser.

How can I do it?

There is a section in the guides: Custom CSS or JavaScript. However, I am not a web developer, and I don't see any explanation there of how to do my desired customization.

I have also posted a [question](https://stackoverflow.com/questions/73282631/modifying-content-width-of-the-sphinx-theme-sphinx-book-theme) on SO about this.

**Comment by AakashGfude** [comment]
Hi @quantitative-technologies , the best way to go about this would be to create your CSS file with the desired style rules, and then add them in your _static folder, as shown in  https://sphinx-book-theme.readthedocs.io/en/stable/customize/custom-css.html?highlight=Custom%20CSS .

Unfortunately, there's no easy way at the moment, like, passing the screen width to a config variable, etc. You will require knowledge of CSS to do it.
